library/src/window_impl.cxx

```diff
@@ -26,6 +26,17 @@
 #include <vtkExternalOpenGLRenderWindow.h>
 #endif
 
+#ifdef _WIN32
+#include <Windows.h>
+#include <dwmapi.h>
+
+#ifndef DWMWA_USE_IMMERSIVE_DARK_MODE
+#define DWMWA_USE_IMMERSIVE_DARK_MODE 20
+#endif
+
+constexpr auto IMMERSIVE_DARK_MODE_SUPPORTED_SINCE = 22000;
```

Comment:
Should this be changed to 19041 just like in MicaForEveryone app? windows documentation says 22000 which is win11.

Reply:
19041 is windows 10?

Reply:
Yes

Reply:
I am changing it to 19041 so whenever @mwestphal gets time to verify this can be directly merged